dev/ci/user-overlays/19132-mattam82-reference-arities.sh

```diff
@@ -0,0 +1 @@
+overlay math_classes https://github.com/mattam82/math-classes reference-arities 19132
```

Comment:
note that the bench doesn't use overlays, you need something like https://github.com/coq/coq/pull/19131/files if you want to use this branch of mathclasses in the bench

Reply:
Good to know, thanks!